Soccar [Game-Server]/src/nl/soccar/gameserver/util/CarUtilities.java

```diff
@@ -18,23 +19,26 @@
     private CarUtilities() {
     }
 
-    public static void addCars(GameEngine engine, Map map, Team blue, Team red) {
-        addCars(engine, map, blue);
-        addCars(engine, map, red);
+    public static void addCars(GameWrapper game, Team blue, Team red) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)